cmd/earthly/main.go

```diff
@@ -412,10 +415,26 @@ func newEarthlyApp(ctx context.Context, console conslogging.ConsoleLogger) *eart
 		},
 		&cli.StringFlag{
 			Name:        "buildkit-host",
+			Value:       buildkitd.DockerAddress,
 			EnvVars:     []string{"EARTHLY_BUILDKIT_HOST"},
 			Usage:       wrap("The URL to use for connecting to a buildkit host. ", "If empty, earthly will attempt to start a buildkitd instance via docker run"),
 			Destination: &app.buildkitHost,
 		},
+		&cli.StringFlag{
+			Name:        "debugger-host",
+			EnvVars:     []string{"EARTHLY_DEBUGGER_HOST"},
+			Usage:       wrap("The URL to use for connecting to a debugger host. ", "If empty, earthly use the default debugger port, combined with the desired buildkit host."),
```

Comment:
Nit: The `debugger-port` usage says host + port forms the URL, while this one says it's itself the URL. This should be clarified.

Reply:
This option should have been removed, actually. Keeping it as it was; it should remain in the config file only, I thought it had always been there as I was moving to the dual-URL approach.